doc/questions.md

```diff
@@ -0,0 +1,55 @@
+# ros-get questions
+
+- *1. where do these packages come from??*
+  - Would be the ros-distro story, but needs a good place in the manual.
+  - perhaps an example and a eference to the REP.
+
+- *2. which branch?*
+  - ``ros-get update`` does what wrt branches?
+  - should it do other things with branches too? (make them, remove them, switch between them, push them, fetch/pull them, etc)
+  - *3. are all branches updated?* Assuming you can have some repos using a different branch, are these updated too, or is that configurable, or?
+
+- *4. is this true?*
+  - ``ros-get list`` seems to list the top-level packages?
+  - *5. get other information?? how??*
+    - can it provide other lists (manually installed as required versus installed as dependency, for example)
+    - can you get packages that nobody needs any more?
+
+- ``ros-get remove`` considerations:
+  - *6. really? what if it is needed by some other package??* Can you remove a package that is needed as dependency?
+  - *7. is a package removed automatically by itself when it is not required any more??* is that desired?
+  - *8. if so (if automatically removed), can you avoid that?*
```

Comment:
6. Currently its broken. It should remove top-level packages. 
7. We should have an `autoremove` command which does that. TODO